content/docs/git/ide.md

```diff
@@ -0,0 +1,248 @@
+
+During this course, while working with HTML and Pything we might end up working with an code editor to make easier interacting with files and folder. An editor can be considered the notebook of a programmer. This, with the terminal, will be considered your main tools to reach your goals :D
+
+To install VSCode please download it by clicking on the Download button of the next link ->
+
+[VSCode](https://code.visualstudio.com/)
+
+You can start playing with it if you want, during the class we will see how to work with it.
+
+### 2. Review exercises from Tuesday (21/September)
+
+Review the content from last class. Make sure that have completed the prerequisites from last class, including that your email is verified on GitHub. If you get stuck or feel confused about anything, no worries! Please write down any questions you have so that we can go over these in class.
+
+{{< button "../git_and_github/" "Review Previous Session: 21/September" >}}
+
+---
+## Class Curriculum
+
+| Section content                          | Expected time (mins) | Pre - Requirements |
+| ---------------------------------------- | -------------------- | ------------------ |
+| Little quiz with Kahoot  | 10        | ❌             |
+| Recap of what we did last class + questions? | 10 minutes            |❌                 |
+| Learning Markdown | 20 minutes        |     ❌         |
+| Class break                              | 7-10 minutes           | ❌                 |
+| Creating our own Journal in Github                 | 45 minutes             | VSCode installed  |
+| Real life example                 | 10 minutes             | VSCode installed  |
+
+## Lesson Goals
+
+- Know how to use an editor and move around
+- What is a Markdown file
+
+## Learning Markdown
+
+Markdown is a way to write content for the web. It’s written in what people like to call “plaintext”, which is exactly the sort of text you’re used to writing and seeing. Plaintext is just the regular alphabet, with a few familiar symbols, like asterisks ( * ) and backticks ( ` ).
+
+Unlike cumbersome word processing applications, text written in Markdown can be easily shared between computers, mobile phones, and people. It’s quickly becoming the writing standard for academics, scientists, writers, and many more. Websites like GitHub and reddit use Markdown to style their comments.
+
+To get started with the tutorial let´s jump here:
+
+[MARKDOWN TUTORIAL](https://www.markdowntutorial.com/)
+
+## Creating your own journal!
+
+### 0. Command Line cheat sheet
+
+{{< tip >}}
+Remember that you can look up what a command does by using the `man` (manual-pages) command on Mac and Linux, or `help` command on Windows cmd. For example: `man echo` would give you information on what the `echo` command does and how to use it.
+{{< /tip >}}
+
+| command | what it does                                                                                                                               |
```

Comment:
I made a few changes/additions to the cheat sheet in the 21-Sept session (added notes about a couple commands + some additional explanation) if you want to copy that one in to keep it the same! :)

Reply:
@eelanagaraj I can't seem to find it there 🤔